pkg/generate/config/config.go

```diff
@@ -158,6 +158,9 @@ type AdditionalFieldConfig struct {
 	// TargetName refers to the name that will be used instead of
 	// SourceName in the status.
 	TargetName string `json:"target_name,omitempty"`
+	// PrintableColumn determines whether the field should be included in the
+	// AdditionalPrinterColumns list.
+	PrintableColumn bool `json:"printable_column,omitempty"`
```

Comment:
Looking more at this, I am struck by the fact that we have an `AdditionalFieldConfig` and a `FieldConfig` struct in the generator config. These really should be combined so that we have a single structure that describes options for a Field that appears in a CR (Spec, Status, or Foobar)

Reply:
The default is that it would not add the field as an "AdditionalPrinterColumn". That is, when you run a `kubectl get`, that field would not be added. The default for all CRDs is that the default `kubectl get` response is only the metadata name.